packages/platform-core/src/usage-data/usage_data_emitter.ts

```diff
@@ -86,6 +86,9 @@ export class DefaultUsageDataEmitter implements UsageDataEmitter {
       codePathDurations: this.translateMetricsToUsageData(options.metrics),
       input: this.translateDimensionsToUsageData(options.dimensions),
       isCi: isCI,
+      flowReport: {
+        runtime: process.env.npm_config_user_agent,
+      },
```

Comment:
This is a compromise. I.e. using an unused field without modifying schema. (like we do for some other pieces of data).